### PR TITLE
Update 0 3 7

### PR DESCRIPTION
**Pull request by n-gao**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by n-gao** [comment]
This depends on https://github.com/conda-forge/sacred-feedstock/pull/1

**Comment by siboehm** [comment]
Triggered a re-run, let's hope it'll pass

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @siboehm
\ No newline at end of file
+* @n-gao @siboehm
```
Comment (siboehm) [inline .github/CODEOWNERS]:
```suggestion
* @siboehm
```
let's do this in a different PR

File: recipe/meta.yaml [hunk 2]
```diff
@@ -51,3 +52,4 @@ about:
 extra:
   recipe-maintainers:
     - siboehm
+    - n-gao
```
Comment (siboehm) [inline recipe/meta.yaml]:
```suggestion
```
ditto

File: .github/CODEOWNERS (hunk 1 above)
Comment (siboehm) [reply .github/CODEOWNERS]: actually I already did it in #12, see also https://conda-forge.org/docs/maintainer/updating_pkgs.html#updating-the-maintainer-list

**Review by siboehm — CHANGES_REQUESTED** [review]
